src/api.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://jennifers-first-web-service.onrender.com/api",
+});
+
+export const getArticles = () => {
+    return api.get(`/articles`).then(({ data }) => {
+            return data;
+          });
+        };
+
+
+// export const getBasket = () => {
+//   return api.get(`/users/Ant/basket`).then(({ data }) => {
+//     return data;
+//   });
+// };
+
+// export const deleteItemFromBasket = (item_id) => {
+//   return api.delete(`/users/Ant/basket/${item_id}`).then(({ data }) => {
+//     return data;
+//   });
+// };
```

Comment:
This doesnt look like code relevant to this ticket or your api so I am not sure why it has been pushed up to github